lib/active_admin/reform/engine.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails/engine'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.